src/runtime_src/core/tools/common/tests/TestVerify.cpp

```diff
@@ -16,7 +17,15 @@ TestVerify::TestVerify()
 boost::property_tree::ptree
 TestVerify::run(std::shared_ptr<xrt_core::device> dev)
 {
-  boost::property_tree::ptree ptree = get_test_header();
-  runTestCase(dev, "22_verify.py", ptree);
+  auto device_name = xrt_core::device_query_default<xrt_core::query::rom_vbnv>(dev, "");
+  boost::property_tree::ptree ptree;
+  if(device_name.find("IPU") != std::string::npos) {
```

Comment:
```suggestion
  if (device_name.find("IPU") != std::string::npos) {
```
@stsoe I realize there is no `.clang-format` in this project.
Start with a top-level minimalistic one everybody can agree on and add later some more precise per sub-project?

Reply:
fixed